website/docs/reference/sdks/flutter-proxy.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Flutter Proxy SDK
+---
+
+This guide shows you how to use feature toggles in a Flutter app with the [Unleash Proxy](../unleash-proxy.md) and the [Unleash front-end API](../front-end-api). You can also check out the source code for the [Flutter Proxy SDK](https://github.com/unleash/unleash_proxy_client_flutter) on GitHub for more details around the SDK.
```

Comment:
All intra-doc links should include file extensions. From [docusaurus](https://docusaurus.io/docs/markdown-features/links):

> Using relative file paths (_with_ .md extensions) instead of relative URL links provides the following benefits:
>
>    Links will keep working on the GitHub interface and many Markdown editors
>    You can customize the files' slugs without having to update all the links
>    Moving files around the folders can be tracked by your editor, and some editors may automatically update file links
>    A [versioned doc](https://docusaurus.io/docs/versioning) will link to another doc of the exact same version
>    Relative URL links are very likely to break if you update the [trailingSlash config](https://docusaurus.io/docs/api/docusaurus-config#trailingSlash)

```suggestion
This guide shows you how to use feature toggles in a Flutter app with the [Unleash Proxy](../unleash-proxy.md) and the [Unleash front-end API](../front-end-api.md). You can also check out the source code for the [Flutter Proxy SDK](https://github.com/unleash/unleash_proxy_client_flutter) on GitHub for more details around the SDK.
```

Reply:
This was copy paste from the javascript-browser.md

Reply:
I also fixed the javascript-browser.md with the same suggestion